accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -0,0 +1,475 @@
+    - `OpenExisting(String, MemoryMappedFileRights, HandleInheritability)`
+
+### System.IO.Pipes
+
+* **NamedPipeClientStream**
+    - `get_NumberOfServerInstances()`
+* **NamedPipeServerStream**
+    - `RunAsClient(PipeStreamImpersonationWorker)`
+* **PipeStream**
+    - `set_ReadMode(PipeTransmissionMode)`
+    - `WaitForPipeDrain()`
+* **PipeTransmissionMode**
+    - `Message`
+
+### System.IO.Ports
+
+> **OPEN** Should we mark the entire type/assembly?
+
+* **SerialDataReceivedEventArgs**
+    - `get_EventType()`
+* **SerialErrorReceivedEventArgs**
+    - `get_EventType()`
+* **SerialPinChangedEventArgs**
+    - `get_EventType()`
+* **SerialPort**
+    - `.ctor()`
+    - `.ctor(IContainer)`
+    - `.ctor(String)`
+    - `.ctor(String, Int32)`
+    - `.ctor(String, Int32, Parity)`
+    - `.ctor(String, Int32, Parity, Int32)`
+    - `.ctor(String, Int32, Parity, Int32, StopBits)`
+    - `Close()`
+    - `DiscardInBuffer()`
+    - `DiscardOutBuffer()`
+    - `get_BaseStream()`
+    - `get_BaudRate()`
+    - `get_BreakState()`
+    - `get_BytesToRead()`
+    - `get_BytesToWrite()`
+    - `get_CDHolding()`
+    - `get_CtsHolding()`
+    - `get_DataBits()`
+    - `get_DiscardNull()`
+    - `get_DsrHolding()`
+    - `get_DtrEnable()`
+    - `get_Encoding()`
+    - `get_Handshake()`
+    - `get_IsOpen()`
+    - `get_NewLine()`
+    - `get_Parity()`
+    - `get_ParityReplace()`
+    - `get_PortName()`
+    - `get_ReadBufferSize()`
+    - `get_ReadTimeout()`
+    - `get_ReceivedBytesThreshold()`
+    - `get_RtsEnable()`
+    - `get_StopBits()`
+    - `get_WriteBufferSize()`
+    - `get_WriteTimeout()`
+    - `GetPortNames()`
+    - `Open()`
+    - `Read(Byte[], Int32, Int32)`
+    - `Read(Char[], Int32, Int32)`
+    - `ReadByte()`
+    - `ReadChar()`
+    - `ReadExisting()`
+    - `ReadLine()`
+    - `ReadTo(String)`
+    - `set_BaudRate(Int32)`
+    - `set_BreakState(Boolean)`
+    - `set_DataBits(Int32)`
+    - `set_DiscardNull(Boolean)`
+    - `set_DtrEnable(Boolean)`
+    - `set_Encoding(Encoding)`
+    - `set_Handshake(Handshake)`
+    - `set_NewLine(String)`
+    - `set_Parity(Parity)`
+    - `set_ParityReplace(Byte)`
+    - `set_PortName(String)`
+    - `set_ReadBufferSize(Int32)`
+    - `set_ReadTimeout(Int32)`
+    - `set_ReceivedBytesThreshold(Int32)`
+    - `set_RtsEnable(Boolean)`
+    - `set_StopBits(StopBits)`
+    - `set_WriteBufferSize(Int32)`
+    - `set_WriteTimeout(Int32)`
+    - `Write(Byte[], Int32, Int32)`
+    - `Write(Char[], Int32, Int32)`
+    - `Write(String)`
+    - `WriteLine(String)`
+
+### System.Net
+
+* **HttpListenerTimeoutManager**
+    - `set_EntityBody(TimeSpan)`
+    - `set_HeaderWait(TimeSpan)`
+    - `set_MinSendBytesPerSecond(Int64)`
+    - `set_RequestQueue(TimeSpan)`
+* **IPEndPoint**
+    - `Create(SocketAddress)`
+* **SocketAddress**
+    - `.ctor(AddressFamily)`
+    - `.ctor(AddressFamily, Int32)`
+    - `get_Family()`
+    - `ToString()`
+
+### System.Net.Sockets
+
+* **Socket**
```

Comment:
most of these APIs don't sound like they are Windows-specific?

Reply:
They're not.